lib/stsci/tools/fileutil.py

```diff
@@ -756,7 +756,7 @@ def openImage(filename, mode='readonly', memmap=False, writefits=True,
                 fimg.close()
                 del fimg
                 # Image re-written as MEF, now it needs its WCS updated
-                updatewcs.updatewcs(fitsname)
+                #updatewcs.updatewcs(fitsname)
```

Comment:
Will this break any existing functionality outside of `drizzlepac`?

Reply:
Not from the testing that I have done so far.